File: Even_odd.go
package main

import "fmt"

func main(){
	var num int;
	fmt.Printf("Enter the number:")
	fmt.Scanln(&num);
	if (num % 2 == 0){
            fmt.Printf("Even number");
    }else
	{
		fmt.Printf("odd number");
    }
}

File: Even_odd.java
import java.util.Scanner;

public class Even_odd {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number:");
        int n  = in.nextInt();
        if (n % 2 == 0)
        {
            System.out.println("Even number");
        }
        else
        {
            System.out.println("odd number");
        }
    }
}
